templates/compose/frontend.yaml.erb

```diff
@@ -7,17 +7,19 @@ services:
     volumes:
       - ../config/frontend/env.js:/usr/share/nginx/html/config/env.js
     labels:
-      traefik.enable: true
-      traefik.frontend.rule: "PathPrefix:/;Host:<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>"
-      traefik.port: 3000
+      - "traefik.http.routers.frontend.rule=Host(`<%= @config['app']['subdomain'] %>.<%= @config['app']['domain'] %>`) && PathPrefix(`/`)"
```

Comment:
You should put the following to enable TLS
- traefik.http.routers.blog.tls=true
- traefik.http.routers.blog.tls.certresolver=myresolver